src/main/scss/_decoration.scss

```diff
@@ -0,0 +1,5 @@
+// Styles pertaining to commonly used custom styling
+
+.bold {
```

Comment:
Traditionally, I wouldn't recommend using helper style classes like this as you end up with `class="bold italic centered"` where this would better be a class specific to the element(s).